src/Apps/Artist/Routes/Overview/Components/ArtistCareerHighlights.tsx

```diff
@@ -30,6 +30,10 @@ const ArtistCareerHighlights: FC<ArtistCareerHighlightsProps> = ({
       ? [insights.slice(0, mid), insights.slice(mid, insights.length)]
       : [insights]
 
+  const hasEntitiesOrDescription = artist.insights.some(insight => {
```

Comment:
We'll need to make this a bit more safe!

in the top of the render method, we'll want to look and see if there's no artist or insights. This doesn't exit early on null data:

```
if (!artist) {
  return nulll
}
if (!artist.insights.length) {
  return null
} 
```

Ditto over on ArtistInsight.tsx.